sdk/ml/azure-ai-ml/azure/ai/ml/_schema/_feature_set/materialization_settings_schema.py

```diff
@@ -27,8 +27,8 @@ class MaterializationSettingsSchema(metaclass=PatchedSchemaMeta):
     notification = NestedField(NotificationSchema)
     resource = NestedField(MaterializationComputeResourceSchema)
     spark_configuration = fields.Dict()
-    offline_enabled = fields.Boolean(required=True, allow_none=False)
-    online_enabled = fields.Boolean(required=True, allow_none=False)
+    offline_enabled = fields.Boolean()
```

Comment:
Please add some test to reflect this change

Reply:
Added UT